src/api-serverless/src/deploy/deploy.routes.ts

```
@@ -1001,6 +1036,16 @@ deployRoutes.post('/release-bus-v2/authorize', async (req, res) => {
     service: string | null;
     expected_sha: string;
     artifact_digest: string | null;
+    source_ref: string | null;
+    reuse_artifact_run_id: string | null;
+    reuse_artifact_name: string | null;
+    reuse_artifact_digest: string | null;
+    candidate_evidence_mode:
+      | 'legacy-whole-train'
+      | 'strict-single'
+      | 'strict-aggregate'
+      | null;
```

ESLint

[error] Replace `|·'legacy-whole-train'⏎······|·'strict-single'⏎······|·'strict-aggregate'⏎·····` with `'legacy-whole-train'·|·'strict-single'·|·'strict-aggregate'`

(prettier/prettier)